### DISCUSSION TITLE
OpenAPI Block with 'Try it out'

### DISCUSSION THREAD

**Discussion by kongmHUB** [GitbookIO/community#320] [Feature Requests]
Is there any chance will let OpenAPI Block have a 'Try it out' function to test api in the feature version?

**Comment by addisonschultz** [comment]
Definitely a great feature request - we've got this one in our backlog for us to dive deeper into in the future 💪

**Reply by sarahf16** [reply]
This would be something we would find very useful too - and something are competitors are able to offer.

**Reply by Prashant-Vandana** [reply]
We are also looking for this feature. Please consider it on priority.

**Comment by RanRegen** [comment]
I'll add that the API reference is one of the major drawbacks compared to readme.com.
They have a completely different look for it with try-me and more.

I'll add that I'm looking into GitBook due to the Bi-directional Git integration, so I wonder how you can solve the API reference for a Git synced space - This will be awesome!

**Reply by fiz-jupico** [reply]
We are in the process of switching to ReadMe for this exact reason...the API reference features are seriously lacking in my opinion, which is odd considering that Git is all about devs. Instead of adding more integrations for product demo platforms, maybe they should prioritize more dev focused features. Just my 2 cents, but it has been frustrating.

**Comment by Prashant-Vandana** [comment]
Team GitBook, 

I request you to kindly work towards the implementation of the "Try It" feature on a high-priority basis. It's a pretty basic feature provided by most of the API documentation portals. We are facing a lot of issues due to the lack of this feature and are forced to go to Postman for API testing.

**Comment by aikoii** [comment]
Hi all 👋! Our OpenAPI Block now features a 'Test it' button, powered by Scalar, allowing live API testing right from your docs. Visit [our documentation](https://docs.gitbook.com/content-editor/blocks/openapi) for more info. Happy testing!
